_posts/dev/2014-08-02-kura-dio-apis.md

```diff
@@ -129,6 +131,199 @@ button.setInputListener(new PinListener(){
 });
 
 ```
+### Use GPIO Pin with GPIOService for a switch a Led. 
+
+This example was developed for a **Raspberry Pi 3**.
+
+Inside the **jdk.dio.properties** file:
+
+```
+...
+6 = deviceType: gpio.GPIOPin, pinNumber:6, name:GPIO06
+...
+```
+
+Then in the code:
+
+```java
+
+private GPIOService myservice;
+
+KuraGPIOPin pin = this.myservice.getPinByTerminal(6);
+
+// to allow pin configuration use
+//pin = this.myservice.getPinByTerminal(this.options.isConfigPin());
+
+``` 
+In a **component.xml** enter a new service. Select a **Service** and in a **Referenced Service** open **Add** for add  the GPIOService. Select the **GPIOService** and use edit button for set in the bind field to **bindGPIOService** and set the unbind filed to **unbindGPIOService**.
+Created methods bind and unbind in a principale software.
```

Comment:
Please review these sentences. They are not clear and. Please describe what do they do. For example, for the bind/unbind methods you can say that are used to bind and unbind Kura's gpioService.